theme/src/resources/templates/home_page.hbs

```diff
@@ -352,6 +352,14 @@
 			}
 		}
 
+		const homepage = document.querySelector('.homepage');
+
+		if (homepage) {
+			const header = document.getElementById('header');
+
+			header.classList.add('hide-header');
```

Comment:
You should check to make sure the `header` DOM node is returned before doing anything with it.